_alerts/2017-10-09-segwit2x-safety.md

```diff
@@ -0,0 +1,134 @@
+sources of Bitcoin news such as this website, [/r/bitcoin](https://www.reddit.com/r/Bitcoin/), and the [Bitcoin Forum](https://bitcointalk.org)
+to make sure that it's safe to continue using your particular wallet. As a general rule, you should not perform any transactions
+during and shortly after the hard fork.
+
+## After the fork
+
+Blocks may be slower shortly after the hard fork and your transactions will take longer to confirm. You
+will have to pay a higher transaction fee if your transaction is high priority and you want it confirmed
+in the next block. You should try to get any important transactions done a few days before the hard fork [date](http://bashco.github.io/2x_Countdown/).
+A lot of websites and programs will likely pop up around the time of the hard fork offering to split your Bitcoin
+so you can have access to both your BTC and to the new altcoin created by this contentious hard fork, you should
+exercise caution as many of these services are likely to be scams trying to steal your bitcoin. As the
+hard fork has very little replay protection, most transactions you perform on the Bitcoin network will also
+be valid on the hard fork network. You should keep this in mind when transacting.
+
+Some services may continue to present and name the altcoin created by this contentious hard
+fork as Bitcoin. Users should be aware that this new currency is not Bitcoin. Bitcoin can only
+be changed with the overwhelming consensus of the entire Bitcoin community of individuals,
+miners, developers and economy. Even when there is overwhelming consensus, unless in an
+emergency, a hard fork should have at least a year notice period to give enough time for users to upgrade. This
+hard fork being adopted by the signatories of this agreement achieves none of these things. It is a rushed
+and hasty upgrade which only has minority community support and has been thoroughly rejected by users and the technical community.
+As this new altcoin is not Bitcoin, Bitcoin.org will not list wallets and services that present
+this altcoin as "Bitcoin" or "BTC" to their users after the hard fork. These services have exited
+the Bitcoin system in a contentious manner and therefore are no longer useful to our visitors.
+
+The following companies and services have pledged to adopt the contentious hard fork:
+
+## Wallets
+
++ Abra (United States)
++ Bitcoin.com (St. Kitts & Nevis)
++ **BitPay (United States)**
++ BitPesa (Kenya)
++ BitOasis (United Arab Emirates)
++ **Blockchain.info (UK)**
++ **BTC.com (China)**
++ Circle (United States)
++ **Coinbase (United States)**
++ Coins.ph (Phillipines)
++ GoCoin (Isle of Man)
++ Jaxx (Canada)
++ Luno (Singapore)
++ Ripio (Argentina)
++ Unocoin (India)
++ **Xapo (United States)**
+
+## Exchanges
+
++ ANX (Hong Kong)
++ Bitex (Argentina)
++ bitFlyer (Japan)
++ Bitso (Mexico)
++ BTCC (China)
++ BTER.com (China)
++ **Coinbase (United States)**
++ Coins.ph (Phillipines)
++ CryptoFacilities (UK)
```

Comment:
CryptoFacilities has possibly withdrawn: https://www.reddit.com/r/Bitcoin/comments/740son/nya_signer_uk_company_crypto_facilities_pulls/. Followup to confirm this may be needed.